src/tools/crawling_tools.py

```
@@ -1038,3 +1354,110 @@ async def _update_sources(ctx: Context, source_content_map: Dict[str, str], sour
+            }
+        elif current_section:
+            current_section["content"] += line + "\n"
+    
+    # Add final section
+    if current_section:
+        sections.append(current_section)
+    
+    return sections
+
+
+def _extract_api_mentions(content: str) -> List[str]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)